deps/prebuilt/include/TAEF/WexTestClass.h

```diff
@@ -587,17 +622,25 @@ namespace WEX
 #pragma section("testdata$g_TCM", read)
 #pragma section("testdata$h_TMM", read)
 
+#define TAEF_ATTRIBUTE_SECTION(sectionName)                 __declspec(allocate(sectionName))
+
 // The IntelliSense compiler reports errors when trying to parse 'TAEF_PIN_FUNCTION_SYMBOL', so it should be skipped.
 #if defined(__INTELLISENSE__)
-# define TAEF_PIN_FUNCTION_SYMBOL
+#define TAEF_PIN_FUNCTION_SYMBOL
 #else
+
 /// \internal
 /// <summary> Instruct the compiler not to remove the function from the obj file during the optimization phase </summary>
-# define TAEF_PIN_FUNCTION_SYMBOL                 __pragma (comment(linker, "/include:" __FUNCDNAME__))
+#if !defined(__clang__)
+#define TAEF_PIN_FUNCTION_SYMBOL                 __pragma (comment(linker, "/include:" __FUNCDNAME__))
+#else
+#define TAEF_PIN_FUNCTION_SYMBOL                 __attribute__((used)) 
+#endif
+    
 #endif
 
 /// \internal
-/// <summary> Instruct oacr to ignore "static initializer causes copy on write pages due to member function pointers" warning
+/// <summary> Instruct oa cr to ignore "static initializer causes copy on write pages due to member function pointers" warning
```

Comment:
oops

---
In reply to: [92519294](https://github.com/Microsoft/WinObjC/pull/1523#discussion_r92519294) [](ancestors = 92519294)